tests/queries/test_explain.py

```diff
@@ -90,6 +90,9 @@ def test_postgres_options(self):
         ]
         if connection.features.is_postgresql_16:
             test_options.append({"generic_plan": True})
+        if connection.features.is_postgresql_17:
+            test_options.append({"memory": True})
+            test_options.append({"serialize": True})
```

Comment:
These options should work:
```suggestion
            test_options.append({"serialize": "TEXT"})
            test_options.append({"serialize": "text"})
            test_options.append({"serialize": "BINARY"})
            test_options.append({"serialize": "binary"})
```

Reply:
Likewise for here :)